#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_remove_postproc_support_from_modules-codec-ffmpeg-postprocess.c.dpatch by Daniel T Chen <crimsun@fungus.sh.nu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Hack around libpostproc-dev/multiverse crackfulness by removing 
## DP: postproc support from modules/codec/ffmpeg/postprocess.c

@DPATCH@

--- vlc/modules/codec/ffmpeg/postprocess.c	2005-10-11 03:07:19.000000000 -0700
+++ vlc/modules/codec/ffmpeg/postprocess.c	2005-10-11 03:08:31.000000000 -0700
@@ -35,7 +35,7 @@
 
 #include "ffmpeg.h"
 
-#ifdef LIBAVCODEC_PP
+#if 0
 
 #ifdef HAVE_POSTPROC_POSTPROCESS_H
 #   include <postproc/postprocess.h>
